llvm/include/llvm/IR/IRBuilder.h

```diff
@@ -757,9 +733,7 @@ class IRBuilderBase {
   /// and noalias tags.
   CallInst *CreateElementUnorderedAtomicMemMove(
       Value *Dst, Align DstAlign, Value *Src, Align SrcAlign, Value *Size,
-      uint32_t ElementSize, MDNode *TBAATag = nullptr,
```

Comment:
It looks like the doc comment needs to be updated now, probably for all declarations.